app/javascript/mastodon/features/status/index.js

```diff
@@ -65,10 +67,19 @@ export default class Status extends ImmutablePureComponent {
     intl: PropTypes.object.isRequired,
   };
 
+  constructor (props) {
```

Comment:
`context` should also be passed as an argument.

```js
constructor (props, context) {
  super(props, context);
}
```

Or, it is recommended to assign state without using constructor.

```js
export default class Status extends ImmutablePureComponent {
  static propTypes = {};

  state = {};

  componentWillMount () {}
}
```